styles/globals.css

```diff
@@ -0,0 +1,11 @@
+body {
+  background-image: radial-gradient(circle at center, #decba488, #2bb0ed66) !important;
```

Comment:
Did you try other positionings on the gradient? Wondering if it might look better outside of the center